sdk/maps/Azure.ResourceManager.Maps/CHANGELOG.md

```diff
@@ -9,9 +9,7 @@
 
 ### Breaking Changes
 
-### Bugs Fixed
-
-### Other Changes
+- Upgrade Maps ARM API version to `2023-06-01`
```

Comment:
version bump is `Features Added` instead of `Breaking Changes`.

Reply:
Moved to `Features Added`